### ISSUE TITLE
Q: Certificate replacement after renewal

### ISSUE THREAD

**Issue by alex-makrshakr** [rmbolger/Posh-ACME#163]
Hi there, 

Pretty impressed from the work. 

I have one question: my understanding is that, when `-Install` is passed as an argument or `Install = true` is specified in the configuration file, Posh-ACME automatically store the newly obtained certificate in the Windows Cert Store. 

However, is it possible to remove the old and about-to-expire certificate automatically? 

Thanks, 
Alex

**Comment by rmbolger** [comment]
Hi Alex. Thanks for the kind words!

Your understanding of the Install functionality is correct. However, the module doesn't support uninstalling the old certificate because that would likely break things that are currently dependent on the old one. Even for the common case like IIS, just installing the new certificate into the Windows cert store doesn't automatically update the certificate binding on the associated site. That deployment action is still something you or your automation script would have to take care of. And only after you've updated everything to use the new certificate should you remove the old one from the Windows cert store.

Put more simply, uninstalling the old cert is out of scope for what this module is intended to do.

**Comment by rmbolger** [comment]
If you weren't aware already, I have a companion module to this one, [Posh-ACME.Deploy](https://github.com/rmbolger/Posh-ACME.Deploy). It's intent is to focus specifically on functions related to deploying the certs you get via Posh-ACME.

Unfortunately, I haven't gotten around to actually releasing a 1.0 version yet. But the functions that are in it seem to work for my purposes so far and most of them include a switch called `-RemoveOldCert`.
